### ISSUE TITLE
change code's color in doc?

### ISSUE THREAD

**Issue by hainm** [dask/dask#348]
hi,

is it possible to change font color of codes in doc website? I find it hard really hard to follow.

http://dask.pydata.org/en/latest/inspect.html

cheers
Hai

**Comment by mrocklin** [comment]
Dask docs currently use the standard sphinx `code-block:: python` directive which handles code highlighting.  Can you suggest an alternative?  

Perhaps this conversation should happen at https://github.com/sphinx-doc/sphinx/ ?

**Comment by hainm** [comment]
hi, how's about pandas sphinx's theme here?

http://pandas.pydata.org/pandas-docs/stable/cookbook.html

by the way, this is only minor (I guess). It's just hard to read `dask`'s code in phone.

**Comment by mrocklin** [comment]
Sure, can I interest you in submitting a pull request with this change?

**Comment by hainm** [comment]
sure

Hai

On Mon, Jun 22, 2015 at 11:00 AM, Matthew Rocklin <notifications@github.com>
wrote:

> Sure, can I interest you in submitting a pull request with this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/dask/issues/348#issuecomment-114141822>.
>



-- 
Hai Nguyen

**Comment by mrocklin** [comment]
Any further interest in this?

**Comment by mrocklin** [comment]
Closing for now.  Please reraise if there is further interest.
